Functions/InputOutput/xASL_io_dcm2nii.m

```diff
@@ -79,34 +79,34 @@
 	%end
 	
 	if ismac && str2num(imPar.dcm2nii_version(1:4))<2014
-        imPar.dcm2nii_version = '20190902'; % mac is incompatible with older versions
+        imPar.dcm2nii_version = '20220720'; % mac is incompatible with older versions
 	end
 
     mricron_path = fullfile(myPath,'External','MRIcron');
 	mricron_version_path = fullfile(mricron_path, imPar.dcm2nii_version);
 	switch computer()
 		case {'PCWIN', 'PCWIN64'}
-			ExePath = fullfile(mricron_version_path,'dcm2nii.exe');
+			ExePath = fullfile(mricron_version_path,'dcm2niix.exe');
 		case 'GLNX86'
-			ExePath = fullfile(mricron_version_path,'dcm2nii-lx32');
+			ExePath = fullfile(mricron_version_path,'dcm2niix-lx32');
 		case 'GLNXA64'
-			ExePath = fullfile(mricron_version_path,'dcm2nii-lx64');
+			ExePath = fullfile(mricron_version_path,'dcm2niix-lx64');
 		case {'MACI','MACI64'}
-			ExePath = fullfile(mricron_version_path,'dcm2nii-osx');
+			ExePath = fullfile(mricron_version_path,'dcm2niix-osx');
 		otherwise
 			error('Unknown computer architecture: %s',computer());
 	end
     [stat, attr] = fileattrib(ExePath);
     if ~stat
-        error('dcm2nii application not found: %s',ExePath);
+        error('dcm2niix application not found: %s',ExePath);

```

Comment:
Same as above. not required to replace all `dcm2nii` to `dcm2niiX`, but fine here in the code.